src/js/edu-benefits/1990e/config/form.js

```diff
@@ -68,9 +68,7 @@ const formConfig = {
           path: 'benefits/eligibility',
           title: 'Benefits eligibility',
           uiSchema: {
-            'view:benefitDescription': {
-              'ui:description': eligibilityDescription
-            },
+            'ui:description': eligibilityDescription,
```

Comment:
We've used them to conditionally show description text; I think that's the only use for empty ones like this one had.